contents/teams/marketing/objectives.mdx

```diff
@@ -1,24 +1,137 @@
+
+### Drive word of mouth through improving support (Joe)
+
+**Rationale:** Support processes need optimizing to ensure all users have a great support experience. Good support = better word-of-mouth growth.
+
+**Contributors:** Joe working with Sales & CS teams
+
+**What we'll ship:** LINK TO ISSUE HERE
+
+**We'll know we're successful when:** Support are hitting their SLAs and we're consistently getting great feedback from users about our support.
+
+### Supporting CS in their sales-led activities (Joe)
+
+**Rationale:** TBD
+
+**Contributors:** Joe working with Sales & CS teams
+
+**What we'll ship:** TBD
+
+**We'll know we're successful when:** TBD
+
+### Become default analytics tool for new AI startups (Lior)
+
+**Rationale:** Our strategy is to get in first and AI startups are generally engineering-led, so strongly within our ICP. A significant portion of recent YC batches are AI startups and they have specific needs we need to cater to.
+
+**Contributors:** Lior
+
+**What we'll ship:** [See issue](https://github.com/PostHog/meta/issues/190)
+
+**We'll know we're successful when:** We have 50 customers using the Langfuse integration and strong usage among AI startups in upcoming YC batches.
+
+### Finish alternatives guides (Andy)
+
+**Rationale:** We know these improve visibility and drive signups. We have a long list of guides we can ship and a proven formula we can repeat.
+
+**Owner:** Andy
+**Contributors:** Andy, Ian, Lior, James
+
+**What we'll ship:**
+- LINK TO ISSUE
+
+**We'll know we're successful when:** We've shipped core alternatives guides for A/B testing, feature flags, and web analytics alternatives.
+
+### Scale newsletter (James)
+
+**Rationale:** We've successfully reached 10k subs and people like the newsletter. We think it can be a big driver of word of mouth growth if we can scale it further.
+
+**Contributors:** James Temperton, Andy, Ian, Lior, James Hawkins. 
+
+**What we'll ship: (WIP)**
+- Transition to 1 new issue per week
+- Improve content promotion on Twitter and LinkedIn
+- Trade way more recommendations with other newsletters
+- Get at least 1 new (or old) post to go viral
+- Improve visibility on the website
+- Do more issues that use our network
+- Land a big interview
+- Continue newsletter sponsorships
+
+**We'll know we're successful when:** We can consistently ship great newsletters every week while maintaining a strong open rate (45% or more) and subscriber growth is accelerating.
+
+### Level up branding (Lottie)
+
+**Rationale:** People love our brand, but it will go stale if don't refresh and create new things that spark joy and conversation.
+**Contributors:** Lottie working with Website & Docs
+
+**What we'll ship:** [See Lottie's issue](https://github.com/PostHog/meta/issues/191)
+
+**We'll know we're successful when:** People continue to talk about our brand and sharing the awesome new things we ship.
+
+### Ship awesome technical blogs (Ian)
```

Comment:
I feel like this is kind of sidequesty if we want to put it there, especially because I will be the only one working on it. I can own docs updates if that is the case.

Reply:
That sounds great. I didn't feel totally comfortable owning docs as a project, so this makes sense to me.